doc/mesh-adapter-api.txt

```diff
@@ -0,0 +1,137 @@
+BlueZ D-Bus Mesh Adapter API description
+********************************
+
+Mesh Network Hierarchy
+======================
+Service		org.bluez.mesh
+Interface	org.bluez.mesh.AdapterManager1
+Object path	/org/bluez/mesh/adapter
+
+Methods:
+	uint64 token Add(string io_type, dict config)
+
+	    This is method that an application has to call to
+	    add a new mesh adapter with an IO type.
+
+	    The config parameter is a dictionary with configuration
+	    options specific for the chosen IO type. In case of "tcpserver",
+	    mesh adapter with given UUID and NetworkKey can added only once.
+	    When adapter with UUID and NetworkKey is already exists then
+	    AlreadyExists error will be raised. When one of required
+	    configuration options is missing then InvalidArguments
+	    error will be raised.
+
+        string io_type
+
+            This parameter choose which IO type will be used
+            to communicate with an adapter.
+
+            The defined values are: "tcpserver", "silvair"
```

Comment:
If this is to be merged upstream, I believe `silvair` has to be changed.

Reply:
I guess that this will not be merged to the upstream. @michallowasrzechonek-silvair Am I right?

Reply:
I don't see that happening anytime soon.

Reply:
OTOH, I think it's high time we fix the naming. "silvar" is a bit random and definitely too generic.

Question: what other (crazy as they might be) io types can we come up with? Ideas:
 - mgmt, for mesh-specific extensions to Linux mgmt API
 - gatt, for GATT proxy

Reply:
* `stdin` :laughing:

Reply:
jsonstdin :D